src/seth/libexec/seth/seth

```diff
@@ -210,6 +210,14 @@ if [[ $SETH_CHAIN ]]; then
         ~/{.local/share,Library/Application\ Support}/io.parity.ethereum/keys/"$SETH_CHAIN"/
       )
       ;;
+    bsctest)
+      eth_rpc_url=https://data-seed-prebsc-2-s1.binance.org:8545/
```

Comment:
are these urls available without api key?

Reply:
Yes, binance have public RPC nodes. You can check all urls here: https://bscscan.com/apis#rpc